doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -357,21 +344,7 @@ glspl{component}
 
 \end_inset
 
- interact with each other.
- If a 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
-
-
-\backslash
-gls{component}
-\end_layout
-
-\end_inset
-
- calls another 
+ (or bundles) at a time, some internal 
```

Comment:
I wanted to distinguish between internal calls (that happen within the software/software components) and external calls (e.g. a call that will open another software), that are not interesting for us